src/NxEditor.EpdPlugin.csproj

```
@@ -42,12 +42,13 @@
       <Private>false</Private>
       <ExcludeAssets>runtime</ExcludeAssets>
     </PackageReference>
+    <PackageReference Include="BfevLibrary" Version="0.11.3" />
   </ItemGroup>
 
   <Target Name="PostBuild" AfterTargets="PostBuildEvent">
     <!-- Copy into plugins dir on anything that is not Windows, should work for both Linux & macOS -->
-    <Exec Condition="!$([MSBuild]::IsOsPlatform('Windows'))" Command="mkdir -p $HOME/.local/share/nx-editor/plugins/$(TargetName)/" ContinueOnError="WarnAndContinue" />
-    <Exec Condition="!$([MSBuild]::IsOsPlatform('Windows'))" Command="cp -R $(TargetDir)/* $HOME/.local/share/nx-editor/plugins/$(TargetName)/" ContinueOnError="WarnAndContinue" />
+    <Exec Condition="!$([MSBuild]::IsOsPlatform('Windows'))" Command="mkdir -p $HOME/.config/nx-editor/plugins/$(TargetName)/" ContinueOnError="WarnAndContinue" />
+    <Exec Condition="!$([MSBuild]::IsOsPlatform('Windows'))" Command="cp -R $(TargetDir)/* $HOME/.config/nx-editor/plugins/$(TargetName)/" ContinueOnError="WarnAndContinue" />
```

This change is not relevant to the PR and should be discussed in another issue/PR

```suggestion
    <Exec Condition="!$([MSBuild]::IsOsPlatform('Windows'))" Command="mkdir -p $HOME/.local/share/nx-editor/plugins/$(TargetName)/" ContinueOnError="WarnAndContinue" />
    <Exec Condition="!$([MSBuild]::IsOsPlatform('Windows'))" Command="cp -R $(TargetDir)/* $HOME/.local/share/nx-editor/plugins/$(TargetName)/" ContinueOnError="WarnAndContinue" />
```